esp8266-fastled-webserver/include/FSBrowser.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
```

Comment:
SIMPLE FILE REVIEW

This header corresponds to FSBrowser.cpp, and exposes only the portions that are used externally.